src/Upgrade/Infrastructure/Report/Sender/RemoteEndpointJsonReportSender.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+declare(strict_types=1);
+
+namespace Upgrade\Infrastructure\Report\Sender;
+
+use GuzzleHttp\ClientInterface;
+use Symfony\Component\Serializer\SerializerInterface;
+use Upgrade\Application\Provider\ConfigurationProviderInterface;
+use Upgrade\Infrastructure\Report\Dto\ReportDto;
+
+class RemoteEndpointJsonReportSender implements ReportSenderInterface
+{
+    /**
+     * @var \GuzzleHttp\ClientInterface
+     */
+    protected ClientInterface $httpClient;
+
+    /**
+     * @var \Symfony\Component\Serializer\SerializerInterface
+     */
+    protected SerializerInterface $serializer;
+
+    /**
+     * @var \Upgrade\Application\Provider\ConfigurationProviderInterface
+     */
+    protected ConfigurationProviderInterface $configurationProvider;
+
+    /**
+     * @var string
+     */
+    protected string $endpointUrl;
+
+    /**
+     * @var int
+     */
+    protected int $timeOut;
```

Comment:
Technically you should use `$timeout` here and `$timeOut` is wrong.

"Timeout" is a noun meaning a specified period of time allowed for an activity.
"Time out" is a noun meaning a temporary suspension of activity or a break from a task.
"Time-out" is a compound noun formed by combining the two words and means the same thing as "time out".

`$timeOut` according to naming convention means that you use `time out`, which has a different meaning.

Reply:
- [ ] Please rename on `$timeout`. It will have the correct meaning